.circleci/config.yml

```diff
@@ -269,6 +270,17 @@ jobs:
               command:  |
                  yarn publish:package -b $CIRCLE_BRANCH
 
+    tag:
+      docker:
+        - image: circleci/node:12.9.1-browsers
+      steps:
+        - checkout
+        - gh/setup
+        - run:
```

Comment:
Do you need to attach the persisted workspace in here so that the /dist folder is available?

Reply:
You're 100% right, we need the dist folder for this work; this is added now.